src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard-v3/dashboard/dashboard-v3.component.html

```diff
@@ -56,7 +56,11 @@
                      link="/hosts"
                      title="Host"
                      summaryType="simplified"
-                     *ngIf="healthData.hosts != null"></cd-card-row>
+                     *ngIf="healthData.hosts != null"
+                     [dropdownEnabled]="hardwareEnabled"
```

Comment:
i don't think we need this to be sent to the other component. If needed, we can just check the `hardwareHealth`  is empty or not to show the dropdown information
```suggestion
```

Reply:
You are right, after my last frontend changes this is definetly not needed, should have gotten rid of it, thanks Nizam